webrtc.html

```diff
@@ -517,6 +517,10 @@
 
         <ol>
           <li>
+            <p>Let <var>p</var> be a new promise</p>.
+          </li>
+
+          <li>
             <p>Append an object representing the current call being handled
```

Comment:
s/an object/a function/, because you execute it further down, and objects can't be executed.